test/scripts/deploy_with_helm.sh

```diff
@@ -12,13 +13,30 @@ source "${SCRIPT_DIR}"/functions
 IP=$(get_ext_ip)
 
 # Use external getopt for long options
-options=$(getopt -o adh --long only-db,auth,help -n "$0" -- "$@")
+options=$(getopt -o adh --long only-db,db-size:,auth,help -n "$0" -- "$@")
 eval set -- "$options"
 
+usage="[--only-db] [db-size=e2e|small|prod]"
+
 while true; do
   case "$1" in
     -a|--only-db) ONLY_DB="--set flightctl.api.enabled=false --set flightctl.worker.enabled=false --set flightctl.periodic.enabled=false --set flightctl.rabbitmq.enabled=false --set flightctl.kv.enabled=false" ; shift ;;
-    -h|--help) echo "Usage: $0 [--only-db]"; exit 0 ;;
+    -h|--help) echo "Usage: $0 $usage"; exit 0 ;;
+    --db-size)
+      db_size=$2
+      if [ "$db_size" == "e2e" ]; then
+        DB_SIZE_PARAMS=""
+      elif [ "$db_size" == "small" ]; then
+        DB_SIZE_PARAMS="--set db.resources.requests.cpu=1 --set db.resources.requests.memory=1Gi --set db.resources.limits.cpu=8 --set db.resources.limits.memory=64Gi"
+      elif [ "$db_size" == "prod" ]; then
```

Comment:
there is no "10k" version here, and no mention of "prod" on the other side.
let's use.

small-1k
medium-10k